HOWTO.rst

```diff
@@ -3826,6 +3826,13 @@ Steady state
 	collection for checking the steady state job termination criterion. The
 	default is 0.  When the unit is omitted, the value is interpreted in seconds.
 
+.. option:: steadystate_check_interval=time, ss_interval=time
```

Comment:
Please also update the description for `ss_dur` to no longer state that performance will be measured once per second.